.circleci/config.yml

```diff
@@ -13,6 +13,15 @@ references:
         at: ~/grakn
 
 jobs:
+  lint:
+    machine: true
+    working_directory: ~/grakn
+    steps:
+      - checkout
+      - *install_bazel
+      - run: python ./verify.py
```

Comment:
@lolski I agree with that. My concern was that we may delegate more responsibilities to it, in which case `check_for_missing_checkstyle_rule` will be a Python _function_ inside of `verify.py` instead

Reply:
what other responsibility may `verify.py` has, @vmax ?

Reply:
@lolski It may verify that after fixing all checkstyle rules we don't have any rules with `allow_failure = True` (alternatively, it can be done by removing the attribute altogether but this approach is more flexible)